client/components/data/query-post-revision-authors/index.jsx

```diff
@@ -0,0 +1,57 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import shallowEqual from 'react-pure-render/shallowEqual';
+
+/**
+ * Internal dependencies
+ */
+import { requestPostRevisionAuthors } from 'state/posts/revisions/authors/actions';
+
+class QueryPostRevisionAuthors extends Component {
```

Comment:
This was `<QueryUsers />` before, see [client/components/data/query-users/index.jsx](https://github.com/Automattic/wp-calypso/pull/27900/files#diff-981dd05db0366a0a438df3638a64388e).